src/backend/distributed/citus--7.0-8--7.0-9.sql

```diff
@@ -0,0 +1,12 @@
+/* citus--7.0-8--7.0-9 */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION check_distributed_deadlocks()
```

Comment:
Hm, might be interesting for deadlock check isolation test reproducability? We could disable / move to infinity the background check and manually trigger the check in a separate session.